src/main/java/com/spotify/autoscaler/AutoscaleJob.java

```diff
@@ -266,7 +274,8 @@ boolean shouldExponentialBackoff() {
     return false;
   }
 
-  int storageConstraints(final Duration samplingDuration, int desiredNodes, int currentNodes) {
+  private int storageConstraints(
+      final Duration samplingDuration, int desiredNodes, int currentNodes) {
```

Comment:
There's a funny mix of `final` and non-`final` parameters to these methods. I guess it's not your responsibility to fix in this PR, I just noticed.